enterprise/authentication-email-password-setup.md

```diff
@@ -0,0 +1,10 @@
+# Authentication with email and password
+## Private Packagist Enterprise
+
+## Setup
+By default authentication with email and password is disabled, and we recommend setting up one of the OAuth options only if they are available.
```

Comment:
I meant for you to adjust this appropriately, please replace with:

"By default, authentication with email addresses and passwords is disabled. If possible, we recommend setting up authentication with your installation of [GitHub (Enterprise)](./github-integration-setup.md), [Bitbucket.org](./bitbucket-integration-setup.md), [Bitbucket Server (Stash)](./bitbucket-server-integration-setup.md), or [GitLab](./gitlab-integration-setup.md) using OAuth instead."

Reply:
updated